x-pack/plugins/ml/public/cases/register_anomaly_charts_attachment.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiDescriptionList } from '@elastic/eui';
+import type { CasesUiSetup } from '@kbn/cases-plugin/public';
+import type { CoreStart } from '@kbn/core/public';
+import type { PersistableStateAttachmentViewProps } from '@kbn/cases-plugin/public/client/attachment_framework/types';
+import { FIELD_FORMAT_IDS } from '@kbn/field-formats-plugin/common';
+import {
+  ANOMALY_EXPLORER_CHARTS_EMBEDDABLE_TYPE,
+  AnomalyChartsEmbeddableInput,
+  getEmbeddableComponent,
+} from '../embeddables';
+import type { MlStartDependencies } from '../plugin';
+import { PLUGIN_ICON } from '../../common/constants/app';
+
+export function registerAnomalyChartsCasesAttachment(
+  cases: CasesUiSetup,
+  coreStart: CoreStart,
+  pluginStart: MlStartDependencies
+) {
+  cases.attachmentFramework.registerPersistableState({
+    id: ANOMALY_EXPLORER_CHARTS_EMBEDDABLE_TYPE,
+    icon: PLUGIN_ICON,
+    // TODO check where this name is presented
```

Comment:
Is this todo still needed

Reply:
removed in https://github.com/elastic/kibana/pull/139628/commits/27177da03fd0d90857f025a3aaa816f63d56acbf